Marlin/ubl_G29.cpp

```diff
@@ -1489,26 +1615,26 @@
         }
       }
 
-      if (x_max - x_min + 1 < grid_size_G || y_max - y_min + 1 < grid_size_G) {
+      if ((x_max - x_min + 1) < (grid_size) || (y_max - y_min + 1) < (grid_size)) {
```

Comment:
No need to parenthesize a variable. Only macros need to be bounded, because they may contain expressions.